src/Symfony/Component/Serializer/Annotation/Ignore.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Annotation;
+
+/**
+ * Annotation class for @Ignore().
+ *
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD"})
```

Comment:
actually I'm wondering what is the exact behavior if this annotation is added to a getter or setter method? Does it make a difference?

So let's say I have a class like this:

```php
class IgnoredAttributeDummy
{
    private $readIgnoredProp;

    private $writeIgnoredProp;

    /**
     * @Ignore()
     */
    public function getReadIgnoredProp()
    {
        return $this->readIgnoredProp;
    }

    public function setReadIgnoredProp($readIgnoredProp): void
    {
        $this->readIgnoredProp = $readIgnoredProp;
    }

    public function getWriteIgnoredProp()
    {
        return $this->writeIgnoredProp;
    }

    /**
     * @Ignore()
     */
    public function setWriteIgnoredProp($writeIgnoredProp): void
    {
        $this->writeIgnoredProp = $writeIgnoredProp;
    }
}
```

Will `$readIgnoredProp` only be ignored when serializing? Or also when deserializing?

Same for `$writeIgnoredProp`: Only ignored when deserializing or also when serializing?

Reply:
Currently we doesn’t distinguish normalization and denormalization in Symfony. We do in API Platform.
I’m not sure how easy it is to introduce such distinction in Symfony, but I think it is worth it in this case so +1 on my side (it’s not handled yet in this PR).

Reply:
Actually, this a deeper issue than expected. I propose to deal with it in another PR as it as broader implications than just `ignore`.